test/transaction.ts

```diff
@@ -151,7 +159,7 @@ describe('Transaction', () => {
     });
 
     it('should commit', done => {
-      transaction.request_ = config => {
+      transaction.request_ = (config: RequestConfig & {gaxOptions: unknown}) => {
```

Comment:
while we're in here, maybe use `sandbox.stub` for this so we don't need to restate the types?

Reply:
Got it 👍